pages/management/env-vars.md

```diff
@@ -1,74 +1,56 @@
 
-Resin provides two kinds of environment variables: application-wide and per-device.
+## Fleet Service Variables
 
-## Application-wide
+Service variables defined on the fleet level are available to the specified service on any device in that fleet, unless they are redefined with a device service variable of the same name.
 
-Environment variables defined on the application level are available to the code on any device running this application — unless they are _redefined_ with the per-device environment variable of the same name.
+The application dashboard has a *Service Variables* tab containing a list of all fleet service variables.
 
-## Per-device
+To define a new fleet service variable, make sure you are in the *Service Variables* tab, then click the *Add variable* button in the upper-left corner:
 
-Per-device environment variables are available to the code running on the particular device. If both the application and the device have an environment variable _of the same name_, the code on this device will see the value of the _per-device environment variables._ In other words, per-device environment variables redefine (or override) application-wide environment variables of the same name.
+<img src="/img/env-vars/add_application_variable.png" width="40%">
 
-It's important to understand that the device environment variable _doesn't have to_ correspond to some application-wide environment variable.
+A new window will pop up, asking you to select a service from the drop down menu. Once you have selected the service, you can define the name and value for your service variable. Click *Add* to apply to all devices in your fleet that do not already have an identical device service variable defined:
 
-## Managing from the Application Page
+<img src="/img/env-vars/variable_editor.png" width="60%">
 
-![Application Page](/img/env-vars/app.png)
+Your new service variable will show up in the list, where it can easily be edited or removed:
 
-The application page has an "Environment variables" tab containing the list of all application-wide environment variables.
+<img src="/img/env-vars/variable_list.png" width="100%">
 
-Here you can modify or delete any variable. Changes are saved automatically, so you just have to edit the value.
+__Note:__ Deleting a fleet service variable will not delete a device variable of the same name.
 
-Each variable row in the list has an expandable region where you see per-device redefines of this variable, can edit, reset them (the Reset button deletes the redefining per-device variable which automatically makes the application-wide variable seen by the device), or add a new redefine by choosing one of the remaining devices from the drop-down selector.
+If you have already defined service variables at the device level, they will appear in a list below your fleet service variables. You can easily apply a device value to the entire fleet by clicking *Define app-wide*:
```

Comment:
`Define app-wide` -> `Define fleet-wide`

(Note that this hasn't yet been changed on multistaging, but it will)